packages/logger/src/types.ts

```diff
@@ -0,0 +1,99 @@
+import cloneDeep from 'lodash.clonedeep';
```

Comment:
```suggestion
import cloneDeep from 'lodash/clonedeep';
```

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/34fab90d6a1a69d58b44c43d0be32b66836d41df